eva-lib/src/main/java/uk/ac/ebi/eva/lib/filter/VariantEntityRepositoryFilter.java

```diff
@@ -27,13 +27,13 @@
 import java.util.Objects;
 import java.util.stream.Collectors;
 
-public class VariantRepositoryFilter<T> {
+public class VariantEntityRepositoryFilter<T> {
```

Comment:
I think the naming comes as a result of this comment: https://github.com/EBIvariation/eva-ws/pull/28#discussion_r99585667